Core/Object Arts/Dolphin/IDE/Base/HelpImagesCreationTest.cls

```diff
@@ -321,20 +321,20 @@ testSystemFolder
 	(folderPresenter view)
 		lvmSetBkImage: LVBKIMAGEW new;
 		backcolor: (Color 
-					red: 255
-					green: 255
-					blue: 238).
+					red: 41
+					green: 124
+					blue: 182).
 	folderPresenter view saveHelpImageNamed: 'SystemFolderClient'.
 	originalExtent := folderPresenter topShell view extent.
 	folderPresenter topShell view extent: 1000 @ 500.
-	stsTools := folderPresenter list detect: [:each | each description = 'Source Tracking Tools'].
+	"stsTools := folderPresenter list detect: [:each | each description = 'Source Tracking Tools']."
 	samples := folderPresenter list detect: [:each | each description = 'Sample Applications'].
 	additionalTools := folderPresenter list detect: [:each | each description = 'Additional Tools'].
 	"Top level icon images"
 	self saveIconImagesInFolder: folderPresenter.
 	"Source Tracking Tools images"
-	folderPresenter expand: stsTools.
-	self saveIconImagesInFolder: folderPresenter.
+	"folderPresenter expand: stsTools.
```

Comment:
I suggest deleting this section entirely. If the STS tools are every brought back to life, they'd be a separate thing.